ADAL/src/request/ADAuthenticationRequest+Broker.m

```diff
@@ -186,74 +185,82 @@ + (ADAuthenticationResult *)processBrokerResponse:(NSURL *)response
     }
     
     // NSURLComponents resolves some URLs which can't get resolved by NSURL
-    NSURLComponents* components = [NSURLComponents componentsWithURL:response resolvingAgainstBaseURL:NO];
+    NSURLComponents *components = [NSURLComponents componentsWithURL:response resolvingAgainstBaseURL:NO];
     NSString *qp = [components percentEncodedQuery];
     //expect to either response or error and description, AND correlation_id AND hash.
-    NSDictionary* queryParamsMap = [NSDictionary adURLFormDecode:qp];
-    
+    NSDictionary *queryParamsMap = [NSDictionary adURLFormDecode:qp];
+
+    NSString *keychainGroup = resumeDictionary[@"keychain_group"];
+
     if([queryParamsMap valueForKey:OAUTH2_ERROR_DESCRIPTION])
     {
-        return [ADAuthenticationResult resultFromBrokerResponse:queryParamsMap];
+        // In the case where Intune App Protection Policies are required, the broker may send back the Intune MAM Resource token
+        NSMutableDictionary *userInfo = [[NSMutableDictionary alloc] initWithDictionary:queryParamsMap];
```

Comment:
seems the name `userInfo` does not reflect what it is, should we rename it to `brokerResponse`? :)